phpunit/class-gutenberg-rest-global-styles-controller-test.php

```diff
@@ -158,17 +155,29 @@ public function test_update_item() {
 		$this->assertEquals( 'My new global styles title', $data['title']['raw'] );
 	}
 
+	/**
+	 * @doesNotPerformAssertions
+	 */
 	public function test_delete_item() {
-		$this->markTestIncomplete();
+		// Controller does not implement delete_item().
 	}
 
+	/**
+	 * @doesNotPerformAssertions
+	 */
 	public function test_prepare_item() {
-		// TODO: Implement test_prepare_item() method.
-		$this->markTestIncomplete();
+		// Controller does not implement prepare_item().
 	}
 
 	public function test_get_item_schema() {
-		// TODO: Implement test_get_item_schema() method.
-		$this->markTestIncomplete();
+		$request    = new WP_REST_Request( 'OPTIONS', '/wp/v2/global-styles/' . self::$global_styles_id );
```

Comment:
If this test any other(like `test_update_item`) have landed in core, we don't need to have them in GB. They can be removed.

Reply:
We have to implement `abstract` methods otherwise tests will fail with fatal error.

```
Fatal error: Class Gutenberg_REST_Global_Styles_Controller_Test contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (WP_Test_REST_Controller_Testcase::test_prepare_item)
```

Reply:
Yes, remove the code and keep an empty function with a comment like `// Covered by the core.` or whatever. It doesn't make any difference. If in some point we update some functions and we need to update this test, we can get the original version from core.

Reply:
Done ✅